man/man7/zfsprops.7

```diff
@@ -1727,6 +1727,15 @@ Please note that the options are comma-separated, unlike those found in
 This is done to negate the need for quoting, as well as to make parsing
 with scripts easier.
 .Pp
+For
+.Fx ,
+there may be multiple sets of options separated by semicolon(s).
+Each set of options must apply to different hosts or networks and each
+set of options will create a separate line for
+.Xr exports 5 .
+Any semicolon separated option set that consists entirely of whitespace
+will be ignored.
+.Pp
```

Comment:
Missed this - you should mention this is a FreeBSD only feature for now.

Reply:
I have added a sentence to indicate that the semicolons
are used for FreeBSD only, as suggested by tonyhutter.